cuda_phase-retrieval/run.py

```diff
@@ -9,45 +9,49 @@
 
 parser = argparse.ArgumentParser()
 parser.add_argument("--image", help="Image path")
-parser.add_argument("--step", help="number of iterations (default 20)", type=int, default=20)
-parser.add_argument("--beta", help="beta (default 0.8)", type=float, default=0.8)
-parser.add_argument("--type", help="Hybrid, Input-Ouput, or Output-Output", default="hybrid")
+parser.add_argument("--mask", help="Mask value (default 1). This will automatically create array of the specified value with the same size of the input image", type=float, default=1)
```

Comment:
Please overwork the help sentence. It is hard to understand. Maybe:

"Initial mask value (default value 1.0). The mask is created automatically and has the size of the input image. All values of the mask are initialized with this value."